### DISCUSSION TITLE
Allow to edit "/ender" menu title

### DISCUSSION THREAD

**Discussion by nhanledev** [Zrips/CMI#7327] [Enhancement]
Hi,

Currently CMI only allows us to modify the Ender Chest title of "other players".
I think it should do the same with own Ender Chest.

The reason why I suggest this is to support custom texture pack. My server modified all the lang file to have a nicer title for every vanilla menu, and the /ender command just does not match the design of the whole server.

Thank you.

**Comment by CloudeLecaw** [comment]
you allready got this:

```
  ender:
    help:
      info: '&eOpens players ender chest'
      args: (playerName)
      explanation:
      - cmi.command.ender.preventmodify - prevents inventory modifications
      - cmi.enderedit - allows to edit opened inventory
    info:
      title: '&8[playerDisplayName] Ender Chest'
      feedback: '&eYou are editing &3[playerDisplayName] &eenderchest (&3[offon]&e).'
```
if you open your own then you see your name there..

**Reply by nhanledev** [reply]
Yes we already have other enderchest title, but own encherchest is not. CMI overwrite vanilla enderchest's title and my suggestion is adding a way to modify it.

**Comment by Zrips** [comment]
Will be fixed with next update.
